xla/debug_options_flags.cc

```diff
@@ -203,6 +203,8 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_experimental_pipeline_parallelism_opt_level(
       DebugOptions::PIPELINE_PARALLELISM_OPT_LEVEL_DISABLE);
 
+  opts.set_xla_gpu_experimental_collective_cse_distance_threshold(100);
```

Comment:
Let's set the default to zero for now. This allows you to experiment with the flag, while still ensuring it doesn't break any existing models.

I suspect it may not be feasible to set this to nonzero by default unless we make significant changes, like moving the pass to run post-scheduling and modifying the pass accordingly. But I'm not sure yet.

Reply:
Wouldn't defaulting to 0 disable the pass? Any concerns with setting it to a low value, perhaps 20?

Reply:
I think the pass should be disabled by default. I worry about regressions causing OOMs if not set to zero.